### PR TITLE
added docs for slack criteria creation

### PR DESCRIPTION
**Pull request by mick-roper**
* added a page to describe Slack Alert fucntionality

### CODE REVIEW

**Comment by mperrien** [comment]
@mick-roper I fixed a few typos, and I have a couple of questions/remarks:

- what is the format of the `ChannelURL`? For the "Reliably Team" Slack channel, is it `reliablyteam` or `reliably.slack.com`? I think we should be explicit about it.
- we could also provide an example of the `MessageTemplate`, I guess.

In the Properties section, there are two links for **Selectors** and **ObjectiveResult**, but there linking to `/`. Where should those go?

Also, I'm wondering if this should go in the **How it works** section, or if we should create an **Integrations** sub-section of the Guides.

File: docs/guides/how-it-works/slack-alerts/index.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+Metadata:
+    Labels:
+        service: my awesome service
+        name: error rate is less than 1 percent
+Spec:
+    IndicatorSelector:
+        category: error_rate
+        aws_resource_arn: arn:partition:service:region:account-id:resource-id
+    ObjectivePercent: 1
+    Window: 1h
```
Comment (Lawouach) [inline docs/guides/how-it-works/slack-alerts/index.md:51]: we need to make these lower case

File: docs/guides/how-it-works/slack-alerts/index.md [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+Metadata:
+    Labels:
+        name: my slack alert
+Spec:
+    ObjectiveSelector:
+        service: my awesome service
+        name: error rate is less than 1 percent
+    ChannelURL: abc123
+    MessageTemplate: abc123
+    RemainingPercentThreshold: 1
```
Comment (Lawouach) [inline docs/guides/how-it-works/slack-alerts/index.md:68]: likewise

**Review by Lawouach — CHANGES_REQUESTED** [review]
Just wondered if that one was ready?

File: docs/guides/how-it-works/slack-alerts/index.md (hunk 2 above)
Comment (mick-roper) [reply docs/guides/how-it-works/slack-alerts/index.md]: done

File: docs/guides/how-it-works/slack-alerts/index.md (hunk 1 above)
Comment (mick-roper) [reply docs/guides/how-it-works/slack-alerts/index.md]: done

File: docs/guides/how-it-works/slack-alerts/index.md [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+title: Integrating with Slack
+excerpt: How to bring Slack into your Reliably ecosystem
+categories: ["guides", "how-it-works"]
+status: published
+type: doc
+---
+
+# Integrating with Slack
+
+Reliably supports integration with Slack. It is possible to configure Realibly to send a message to Slack when the budget for a given objective is less than the threshold set in the alert configuration.
```
Comment (ciaranevans) [inline docs/guides/how-it-works/slack-alerts/index.md:11]:
```suggestion
Reliably supports integration with Slack. It is possible to configure Reliably to send a message to Slack when the budget for a given objective is less than the threshold set in the alert configuration.
```

File: docs/reference/entities/slackalertcriteria.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
+---
+title: SlackAlertCriteria
+excerpt: The definition of SlackAlertCriteria
+categories: []
+status: published
+type: overview
+---
+# SlackAlertCriteria
+
+An `SlackAlertCrtieria` declaratively defines the data required to send a message to a Slack channel, and the conditions required for the criteria to be used.
```
Comment (ciaranevans) [inline docs/reference/entities/slackalertcriteria.md:10]:
```suggestion
A `SlackAlertCrtieria` declaratively defines the data required to send a message to a Slack channel, and the conditions required for the criteria to be used.
```

This ones pedantic of me

File: docs/reference/entities/slackalertcriteria.md [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+
+## Structure
+
+```yaml
+apiVersion: reliablyhq.com/v1
```
Comment (ciaranevans) [inline docs/reference/entities/slackalertcriteria.md:15]:
```suggestion
apiVersion: reliably.com/v1
```

**Review by ciaranevans — CHANGES_REQUESTED** [review]

**Review by ciaranevans — APPROVED** [review]

**Pull request merged**